sklearn/compose/_target.py

```diff
@@ -168,6 +168,11 @@ def fit(self, X, y, **fit_params):
         Returns
         -------
         self : object
+
+        .. versionchanged:: 0.22
```

Comment:
Numpydoc won't process this correctly, since it's in the Returns section

I'm not sure whether we need such notes, but they have to appear in a Notes section or in the top description section.

Reply:
I don't think such a bug fix requires that note anyway.